src/theme/Navbar/Links.js

```diff
@@ -1,16 +1,16 @@
-import React from 'react'
-import clsx from 'clsx'
+import React from "react";
```

Comment:
Minor but seems like a huge amount of these changes are simply from mis-aligned lint?

Reply:
yes

Reply:
I reverted all the changes